run.py

```diff
@@ -4,7 +4,7 @@
 import sys
 
 from flask import Flask, g
-
+from config_parser import parse_config
```

Comment:
Which module is this?
I googled and found https://docs.python.org/2/library/configparser.html but Its ConfigParser and not config_parser